src/components/chat/chat-panel.tsx

```diff
@@ -66,12 +66,11 @@ export const ChatPanel: React.FC<IProps> = ({ user, activeNavTab, focusDocument,
   }, [commentsPath, deleteCommentMutation, focusDocument, focusTileId]);
 
   const newCommentCount = unreadComments?.length || 0;
-
   return (
     <div className={`chat-panel ${activeNavTab}`} data-testid="chat-panel">
       <ChatPanelHeader activeNavTab={activeNavTab} newCommentCount={newCommentCount}
                        onCloseChatPanel={onCloseChatPanel} />
-      {focusDocument
+      {focusDocument || focusTileId
```

Comment:
Clearly there's something else going on here that is not directly related to the nav panel. Was this just a bug?